handlers/organization_test.go

```diff
@@ -86,6 +86,21 @@ func TestUnitCreateOrEditOrganization(t *testing.T) {
 		assert.Equal(t, http.StatusBadRequest, rr.Code)
 	})
 
+	t.Run("should return error if org name contains only spaces", func(t *testing.T) {
+		rr := httptest.NewRecorder()
+		handler := http.HandlerFunc(oHandler.CreateOrEditOrganization)
+
+		invalidJson := []byte(`{"name": "   "}`)
+		req, err := http.NewRequestWithContext(ctx, http.MethodPost, "/", bytes.NewReader(invalidJson))
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		handler.ServeHTTP(rr, req)
+
+		assert.Equal(t, http.StatusBadRequest, rr.Code)
+	})
```

Comment:
lets add another test to create a name that looks like "    Abdul    " and assert it gets added as "Abdul"

Reply:
@kevkevinpal Added https://github.com/stakwork/sphinx-tribes/pull/1553